recipe/envs.go

```diff
@@ -0,0 +1,44 @@
+package recipe
+
+import (
+	"os"
+	"strings"
+)
+
+var (
+	recipeEnvVarPrefix = "METEOR_"
+)
+
+func populateData() map[string]string {
+	data := make(map[string]string)
+	for _, envvar := range os.Environ() {
+		keyval := strings.SplitN(envvar, "=", 2) // "sampleKey=sample=Value" returns ["sampleKey", "sample=value"]
+		key := keyval[0]
+		val := keyval[1]
```

Comment:
Should we expand `value` using `os.ExpandEnv` as well?
If someone provides an env like `output=$HOME`, I think value needs to be expanded to work although I am not sure if that is one of the possible use cases users can use.